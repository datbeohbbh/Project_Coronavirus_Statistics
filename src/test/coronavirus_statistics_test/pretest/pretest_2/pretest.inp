10 9
q 1 6 9
q 1 2 11
? 4 2
q 3 6 4
? 4 2
q 1 4 41
q 2 2 11
? 6
? 7