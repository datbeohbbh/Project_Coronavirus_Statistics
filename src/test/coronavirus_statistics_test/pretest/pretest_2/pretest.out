Query ? 4 2 :
The answer is the city with the index 2 
- Information of the city 2 -
-> Number of current patients : 11
-> Has recovered : 0
-> Was dead : 0

Query ? 4 2 :
The answer is the city with the index 2 
- Information of the city 2 -
-> Number of current patients : 11
-> Has recovered : 0
-> Was dead : 0

Query ? 6 :
Currently, no recovery cases have been recorded

Query ? 7 :
The city with the highest death is city with the index 6
Quantity: 4 people

